src/lib/journalctl/entry_presenter.rb

```diff
@@ -50,5 +50,10 @@ def source
     def formatted_time
       timestamp.strftime(TIME_FORMAT)
     end
+
+    # Message string
+    def message
+      entry.message.to_s
```

Comment:
please add also explanation why it is needed. Especially as this is tutorial module, it need explanation why it is needed this way.